src/utils/exportUtils/HtmlExport.tsx

```diff
@@ -161,7 +161,7 @@ export default class HTMLExporter extends Exporter {
                     <div class="mx_MatrixChat_wrapper" aria-hidden="false">
                         <div class="mx_MatrixChat">
                         <main class="mx_RoomView">
-                            <div class="mx_Flex mx_RoomHeader light-panel">
+                            <div class="mx_RoomHeader light-panel">
```

Comment:
Flex class name not showing up in snapshots seems like an issue, you can't differentiate between a `div` and a `Flex` container